packages/components/src/mobile/progress-circle/index.native.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import Svg, { Circle } from 'react-native-svg';
+import Animated from 'react-native-reanimated';
+
+/**
+ * Internal dependencies
+ */
+
+const ProgressCircle = ( { progress, width = 20, strokeWidth = 2 } ) => {
+	const circleSize = width;
+	const r = ( circleSize - strokeWidth ) / 2;
+	const cx = circleSize / 2;
+	const cy = circleSize / 2;
+
+	const { interpolate, multiply } = Animated;
+	const { PI } = Math;
+	const circumference = r * 2 * PI;
+	const α = interpolate( progress, {
+		inputRange: [ 0, 100 ],
+		outputRange: [ 0, PI * 2 ],
+	} );
+
+	const strokeDashoffset = multiply( α, r );
+
+	const AnimatedCircle = Animated.createAnimatedComponent( Circle );
+
+	return (
```

Comment:
I didn't want to hardcode the stroke colors. At first, I tried putting them in a style sheet but it didn't work. Just to verify, I can add a stroke attribute to a style with say `$blue-wordpress` and that should work, right? I am going to re-add the stylesheet tomorrow so the stroke and fill values are stored there.

Reply:
Since `stroke` is a property on its own, you would need to assign it directly. Something like this:
`stroke={ styles.circleProgress.stroke }`
This I think should work